circleguard/comparer.py

```diff
@@ -144,7 +146,35 @@ def _compare_two_replays(replay1, replay2):
         if (Mod.HR in replay1.mods) ^ (Mod.HR in replay2.mods):
             xy1[:, 1] = 384 - xy1[:, 1]
 
-        return Comparer._compute_data_similarity(xy1, xy2)
+        (play_sequence_1, play_sequence_2) = Comparer.convert_to_row_major_form(xy1, xy2)
+
+        NUM_CHUNKS = 20
+
+        # Sectioned into 20 chunks, used to ignore outliers (eg. long replay with breaks is copied, and the cheater cursordances during the break)
+        horizontal_length = play_sequence_1.shape[1] - play_sequence_1.shape[1] % NUM_CHUNKS
+        play_sequence_1_sections = np.hsplit(play_sequence_1[:,:horizontal_length], NUM_CHUNKS)
```

Comment:
after interpolation both replays are equally long so you might be able to include the remainder section in the cross correlation

Reply:
Yeah, I could, you think it's worth it for the slightly increased code complexity?

Reply:
Also, got any feedback around fix chunk sizes vs fixed number of chunks? Based on the sample rate, I think it should be okay for any ranked maps, but who knows.

Reply:
I don't think the code would get very much more complex. I also have no strong opinion on fixed/variable chunks but I am not opposed to it being a default variable with value 20

Reply:
Hmm, might not be worth adding, as you could potentially end up with a slice with just 1 data point, for which this would fail on.

Will parameterize the number of chunks though and default it to 20.